src/Nox.Integration.SqlServer/Nox.Integration.SqlServer.csproj

```diff
@@ -0,0 +1,53 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <TargetFramework>net7.0</TargetFramework>
+        <ImplicitUsings>enable</ImplicitUsings>
+        <Nullable>enable</Nullable>
```

Comment:
you can add at this level for all configurations:
`<TreatWarningsAsErrors>true</TreatWarningsAsErrors>`